source/Engine/iw_engine.cpp

```diff
@@ -276,21 +288,27 @@ namespace IW
 			UI_FormattedBuildNumber = (UI_FormattedBuildNumber_t)build_info_func;
 		}
 
-		if (*(DWORD*)(0x46F590_b + 1) == 0x224149) // [S2_SP64_SHIP 1.25.2244937]
+		if (*(DWORD*)(0x46F590_b + 1) == 0x224149)                               // [S2SP64 1.25.2244937]
 		{
 			Sys_EnterCriticalSection = (Sys_EnterCriticalSection_t)0x5369A0_b;   // [S2SP64 1.25.2244937]
 			Sys_LeaveCriticalSection = (Sys_LeaveCriticalSection_t)0x536A10_b;   // [S2SP64 1.25.2244937]
 
-			cmd_textArray = (CmdText*)0x98DFDB8_b; CRITSECT_CBUF_NUMBER = 193;   // [S2SP64 1.25.2244937]
+			cmd_textArray = (CmdText*)0x98DFDB8_b;                               // [S2SP64 1.25.2244937]
 			cmd_functions = (CmdFunc**)0x98DFBD8_b;                              // [S2SP64 1.25.2244937]
 
+			CRITSECT_CBUF_NUMBER = 193;                                          // [S2SP64 1.25.2244937]
+
 			Dvar_FindVar = (Dvar_FindVar_t)0x520AB0_b;                           // [S2SP64 1.25.2244937]
 			Dvar_ValueToString = (Dvar_ValueToString_t)0x52FD80_b;               // [S2SP64 1.25.2244937]
 
 			loaded_engine_handler.engine_type = ENGINE_S2;
 			loaded_engine_handler.cmd_exec = Cmd_ExecuteText;
 			loaded_engine_handler.dvar_info = Dvar_FetchInformation;
 		}
+		else
+		{
+			throw std::runtime_error("UNSUPPORTED GAME VERSION (S2SP 1.25)");
```

Comment:
No need to throw exception; sdk would still work perfectly fine but wont be able to provide in-depth engine related extra functionallities

Everything else is looking fine to me